### ISSUE TITLE
WCAG: 1.3 Adaptable

### ISSUE THREAD

**Issue by StephDriver** [BirkbeckCTP/janeway#4126]
### Requirement
https://www.w3.org/TR/WCAG22/#adaptable
> Create content that can be presented in different ways (for example simpler layout) without losing information or structure.

#### :x: Info and Relationships 
Pages are not consistently structured such that information and relationships are unclear. Some of this is ours, and some is due to poor underlying structure within the JATS which is user-content. 

#### ✅ Meaningful Sequence 
Layout is divided into blocks, with meaning derived from the block grouping rather than the sequence in which the blocks appear on the page, thus there are no concerns about lack of meaningful sequence beyond those already being addressed by structural issues above.  No work has been identified.

#### :x: Sensory Characteristics
We do not rely on sound, size or orientation to convey information.  We are in the process of upgrading our Captcha to make it more accessible.  This audit found that we have used icons and typographical marks to convey information without providing non-visual alternatives. Colour is being used in the Material theme to show invalid user input.

#### :x: Orientation
The pages are responsive to wide and narrow width screens which reflect portrait and landscape views, reflowing between the two.  There are a few bugs with the header and navigation on narrower screens.

#### ✅ Identify Input Purpose
Information about the user is input in the Janeway Back Office and not in the front end.

### Work to do

Info and Relationships
- #3911
- #3914
- #3915
- #3921
- #3937
- #3946
- #4055
- #4137
- #4200
- #4211
- #4225
- #4227
- #4232
- #4237
- #4242
- #4244
- #4245
- #4246
- #4247

Sensory Characteristics
- #3945
- #4210
- #4223
- #4238
- #4243
- #4230
- #4058
- #4215
- #4200
- #4227
- #3936
- #4268 

Orientation
- #4225 
- #4226
